cmd/executor/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Add test case for no config.

Reply:
Actually, the case you mention covers this error too, so I will make the existing test more general (ie. test a no config situation).